docs/index.html

```diff
@@ -145,29 +145,14 @@ <h3>Map Controls</h3>
     <div class="no-break">
         <h3>Browser Support</h3>
 
-        <h4>Desktop</h4>
-
         <ul>
             <li>Chrome</li>
             <li>Firefox</li>
-            <li>Safari 5+</li>
-            <li>Opera 12+</li>
-            <li>IE 9&ndash;11</li>
+            <li>Safari</li>
             <li>Edge</li>
```

Comment:
What about legacy Edge? It is [end of life](https://learn.microsoft.com/en-us/lifecycle/announcements/m365-ie11-microsoft-edge-legacy) since March 9, 2021.

Reply:
Well, since Microsoft makes no distinction I think we can do the same. Naturally we don't support 'legacy' edge.

Reply:
Edge is now removed from the code

Reply:
The version of Edge that was using the old engine yes, but we still support the Chromium version.